community/cypher/cypher-compiler-3.2/src/main/scala/org/neo4j/cypher/internal/compiler/v3_2/commands/expressions/StringFunctions.scala

```diff
@@ -66,8 +66,14 @@ case class ToLowerFunction(argument: Expression) extends StringFunction(argument
 
 case class ReverseFunction(argument: Expression) extends StringFunction(argument) {
```

Comment:
Since this function no longer applies only to strings, it should no longer be in the StringFunction.scala. Consider pulling it out.